README.md

```diff
@@ -62,14 +62,27 @@ make cluster/prepare/local
 ```
 
 
-* 3scale requires AWS credentials for backups to an S3 bucket. The bucket should have all public access turned off.
+* 3scale requires AWS S3 bucket credentials for storage. The bucket should have all public access turned off.
 Currently this secret (`threescale-blobstorage-<installation-name>`) is created with dummy credentials by the [cloud resource operator](https://github.com/integr8ly/cloud-resource-operator), in the namespace the integreatly operator is deployed into. In order for this feature to work, these credentials should be replaced:
     * _bucketName_: The name of the AWS bucket
     * _bucketRegion_: The AWS region where the bucket has been created
     * _credentialKeyID_: The AWS access key
     * _credentialSecretKey_: The AWS secret key
-   
 
+You can use this command to replace S3 credentials in 3Scale secret:
```

Comment:
why do we need this? I thought the s3 credentials were created by the cloud resources operator now?

Reply:
They will be in production, or in OCP clusters with "credentials minter".
But it doesn't work on OSD clusters that we use for dev now. 

A Secret with credentials will be created, but it will contain placeholder values. And here I describe how override that with valid S3 credentials, just for the testing purposes.